src/events/pollVotationResults.js

```diff
@@ -3,31 +3,32 @@ const { VOTE_POINTS } = require('../config');
 const { translateLanguage } = require('../languages');
 
 const tagIds = VOTE_POINTS.TAG_IDS;
+
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
-    const { embeds, author, client, channelId } = message;
+    try {
+      const { embeds, author, client, channelId } = message;
 
-    if (
-      !author.bot ||
-      !embeds ||
-      !Array.isArray(embeds[0]?.fields) ||
-      embeds[0]?.data?.type !== 'poll_result'
-    ) {
-      return;
-    }
+      if (
+        !author ||
+        author.bot ||
```

Comment:
The line 15 changes the behavior of the conditional, it should be TRUE if the author is a BOT. Here are some suggestions to improve readability of this part:

```suggestion
        !author?.bot ||
```

```suggestion
        (!author ||
        !author.bot) ||
```

Reply:
![image](https://github.com/user-attachments/assets/aaa51836-3d10-4a90-8afe-f510217bf7ce)

Reply:
@heliomar-pena  It won't let me place it that way

Reply:
thanks for confirming!

Reply:
@heliomar-pena 
check